### PR TITLE
Add a "View" menu to Mandelbrot

### PR DESCRIPTION
**Pull request by nico**
It uses the standard actions, so ctrl-=/ctrl--/ctrl-0 can now be used for zooming, just like in ImageViewer.

### CODE REVIEW

File: Userland/Demos/Mandelbrot/Mandelbrot.cpp [hunk 1]
```diff
@@ -209,6 +209,10 @@ class Mandelbrot : public GUI::Frame {
 
     void export_image(String const& export_path);
 
+    enum class Zoom { In,
+        Out };
```
Comment (nico) [inline Userland/Demos/Mandelbrot/Mandelbrot.cpp:213]: wth clang-format

Comment (awesomekling) [reply Userland/Demos/Mandelbrot/Mandelbrot.cpp]: Situation status: not on top.

**Comment by nico** [comment]
(I think the failing JS tests on that one CI bot _might_ be unrelated to this change…)

**Comment by nico** [comment]
Thanks for merging :)

**Pull request merged**